cli/src/alluxio.org/cli/cmd/initiate/copy_dir.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package initiate
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var CopyDir = &CopyDirCommand{}
```

Comment:
since it is not complete, let's not merge it. otherwise it will be confusing to the user who expect it to work